frontends/verilog/verilog_parser.y

```diff
@@ -2325,6 +2364,34 @@ simple_behavioral_stmt:
 		SET_AST_NODE_LOC(node, @2, @5);
 		append_attr(node, $1);
 	} |
+	TOK_INT lvalue '=' delay expr {
```

Comment:
I think it should be possible to reduce duplication between the int/int unsigned/int signed paths? And are there other types that should be considered here like integer and reg?